### PR TITLE
Enhancement library Suggesstion 

### PR DESCRIPTION
**Pull request by ahmed-hamdy90**
This code just enhancement for your library.

- add missing license file for your library (add your name)
- re-place library code under src (source) folder
- create test folder to include future tests of your library
- write missing JSDoc for your library (on the future you can use any Documentation generator)
- re-name and re-write validation file to become Utility class with static helper method instead of create new instance every time you need it (Step 3 on your README)

TODO: 

- your must mention about your library must used under Angular or React not can use direct into pure Javascript project (Does there any way to be work direct into browser), If you target Angular or React preferred to make separated library for every one (or write an example how to use you library on README file)

### CODE REVIEW

File: src/validation.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    return value.toString().trim().length > length;
+}
+
+/**
+ * Check whether given value has been match minmum length
+ * @param  {*} value  value want to check
+ * @param  {number} length minmum length value
+ * @returns {boolean} true if was match, otherwise false
+ */
+function matchMinLength (value, length) => {
```
Comment (Raneen-medhat) [inline src/validation.js:47]: you can not use arrow syntax with **function**  keyword

File: src/validation.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+function isSaudiMobile (value) {
+  return patterns.soudiMobile.test(value.toString().toLowerCase());
+}
+
+/**
+ * Convert given english numbers to arabic numbers
+ * @param  {[]} values  array of values for english numbers
+ * @return {[]} converted arabic numbers
+ */
+function toEnglishNumber (...values) => {
```
Comment (Raneen-medhat) [inline src/validation.js:83]: the same here for arrow function comment

File: .gitigone [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (Raneen-medhat) [inline .gitigone:1]: i think no need to add gitignor file here , do yo have any suggestions to use it ?

**Review by Raneen-medhat — CHANGES_REQUESTED** [review]
Thank you for your contribution, i appreciate your time and effort.
i just left some comments for your pull request, kindly check it
Regards

File: src/validation.js (hunk 1 above)
Comment (ahmed-hamdy90) [reply src/validation.js:47]: yes, copy and paste mistake

File: src/validation.js (hunk 2 above)
Comment (ahmed-hamdy90) [reply src/validation.js:83]: same mistake here https://github.com/Raneen-medhat/validation-js/pull/6#discussion_r372071929

File: .gitigone (hunk 3 above)
Comment (ahmed-hamdy90) [reply .gitigone:1]: Yes now no need for **.gitignore** file (it just ignore node_modules folder only now) as your library not depend on any packages, But on the future may be depend any library at lest dev-dependencies packages when you apply unit test on your **validation.js** file and any one will contribute on library may need to npm install this package to work.

See most popular Libraries [Angular](https://github.com/angular/angular) and [React](https://github.com/facebook/react) Repositories both has **.gitignore** file.

**Comment by ahmed-hamdy90** [comment]
**Little Notices:**
 - Take care about when use Ecmascript 6 arrow function or not to use it, see next useful articles about this:

     -   https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/ (both when use and not use)
     - https://wesbos.com/arrow-function-no-no/ (only not use)
     - https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html (both when use and not use)

**Extra TODO:**

 - You need to setting above Description for library and setting keywords too on Github repository
 - try to add eslint configuration to your library.

**Pull request merged**